### PR TITLE
go-parquet-tools 1.25.10

### PR DESCRIPTION
**Pull request by hangxie**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/g/go-parquet-tools.rb [hunk 1]
```diff
@@ -21,15 +22,15 @@ def install
       -s -w
       -X github.com/hangxie/parquet-tools/cmd.version=v#{version}
       -X github.com/hangxie/parquet-tools/cmd.build=#{time.iso8601}
-      -X github.com/hangxie/parquet-tools/cmd.gitHash=#{tap.user}
-      -X github.com/hangxie/parquet-tools/cmd.source=source
+      -X github.com/hangxie/parquet-tools/cmd.gitHash=#{Utils.git_short_head}
+      -X github.com/hangxie/parquet-tools/cmd.source=Homebrew
```
Comment (chenrui333) [inline Formula/g/go-parquet-tools.rb:26]: If your git tag is immutable, then sha and tag are the same thing, no need to track both tag and sha in the brew build

File: Formula/g/go-parquet-tools.rb [hunk 2]
```diff
@@ -1,8 +1,9 @@
 class GoParquetTools < Formula
   desc "Utility to deal with Parquet data"
   homepage "https://github.com/hangxie/parquet-tools"
-  url "https://github.com/hangxie/parquet-tools/archive/refs/tags/v1.25.9.tar.gz"
-  sha256 "61df81b68b58cfccd6b819bc3edf50e6ecf58e294195167517d71076fb047fd3"
+  url "https://github.com/hangxie/parquet-tools.git",
+    tag:      "v1.25.10",
+    revision: "ef7222f98e756f7ded509fc2e2f63c9a2ddf9e03"
```
Comment (chenrui333) [inline Formula/g/go-parquet-tools.rb:6]:
```suggestion
  url "https://github.com/hangxie/parquet-tools/archive/refs/tags/v1.25.10.tar.gz"
  sha256 "4255a765c295603fdc51b3479ff8d0defbb864971b2e1f732d1278231603b18b"
```

File: Formula/g/go-parquet-tools.rb (hunk 1 above)
Comment (hangxie) [reply Formula/g/go-parquet-tools.rb:25]: Yes tag is meant to be immutable, deprecate for now and I will remove it from next release.

Comment (chenrui333) [reply Formula/g/go-parquet-tools.rb:25]: yeah, commitHash good for dirty build, not good for release build 👍

**Review by chenrui333 — APPROVED** [review]
Thanks!

**Pull request merged**